recipe/meta.yaml

```diff
@@ -24,14 +24,14 @@ requirements:
         - python
         - setuptools
         - numpy
-        - cdat-lite
-        - udunits2
+        - netCDF4
```

Comment:
The build keeps failing because of the upper case package name above. See the logs:

```
BUILD START: cfchecker-3.0.5-py27_0
    (actual version deferred until further download or env creation)
Can't build /recipe_root due to unsatisfiable dependencies:
Package missing in current linux-64 channels: 
  - netCDF4
```